File: Server/JMS-App/src/main/java/rb/org/domain/GameplayTypes.java
package rb.org.domain;

/**
 * Created by bogaa on 6/4/2017.
 */
public enum GameplayTypes {
    Connect("Connect"),
    Movement("Movement"),
    Promotion("Promotion"),
    Turn("Turn"),
    Finish("Finish"),
    Surrender("Surrender"),
    None("None");

    private final String gameplayType;

    private GameplayTypes(final String gameplayType) {
        this.gameplayType = gameplayType;
    }

    @Override
    public String toString() {
        return gameplayType;
    }
}

File: Assets/Scripts/Messages/GameplayTypes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Assets.Scripts.Messages
{
    public enum GameplayTypes
    {
        Connect,
        Disconnect,
        Initialised,
        Movement,
        Promotion,
        Turn,
        Finish,
        Surrender,
        NONE
    }
}
